### DISCUSSION TITLE
[examples/http-axum-apigw-authorizer] How to deploy?

### DISCUSSION THREAD

**Discussion by hffmnn** [aws/aws-lambda-rust-runtime#862] [Q&A] [answered]
Hi,

is there an example of how to deploy `examples/http-axum-apigw-authorizer`? When deploying via `cargo lambda deploy` it only deploys the lambda but no API GW. And then there shouldn't be an authorizer, right?

Did I miss something?

**Comment by bnusunny** [comment, marked as answer]
Yes, this example only creates a Lambda function. You need to create an API Gateway separately. 

There are examples of how to create an API Gateway + Lambda Authorizer + Lambda function. Such as t[his one](https://serverlessland.com/patterns/apigw-lambda-authorizer-basic-cdk) on ServerlessLand.

**Comment by hffmnn** [comment]
Thanks a lot. It works with this example using e.g. `https://github.com/cargo-lambda/cargo-lambda-cdk` for the protected resource
